pkg/pocket/executor/common.go

```diff
@@ -116,3 +125,37 @@ func (e *Executor) TxnRollback() error {
 	}
 	panic("unhandled txn rollback switch")
 }
+
+// since the tiflash cluster is always conn1 in TiFlash abtest, so it is
+// okay to just use single test ddl here.
+func (e *Executor) createTiFlashTableReplica(table string) error {
+	return e.SingleTestExecDDL(fmt.Sprintf("ALTER TABLE %s SET TIFLASH REPLICA 1", table))
+}
+
+func (e *Executor) WaitTiFlashTableSync(table string) error {
+	sql := fmt.Sprintf("SELECT AVAILABLE FROM information_schema.tiflash_replica WHERE TABLE_SCHEMA = '%s' and TABLE_NAME = '%s'",
+		e.dbname, table)
+	for {
+		res, err := e.GetConn().Select(sql)
+		if err != nil {
+			return err
+		}
+		// table doesn't exist
+		if res == nil {
+			return nil
```

Comment:
If the table does not exist, why not wait and check it next time?

Reply:
This is for the case when the generated table name doesn't exist like `s4`, `s5`. Seems they won't be created. So I have to return nil. To be honest, I'm not really sure about this case.

Reply:
`s4`, `s5` will only be generated in select statements by mistake. I find you wait for table sync before read statements, why not just wait for the sync after create table statements?

Reply:
If I don't wait before read, it will produce some error.